contrib/dist/release_checklist_issue_template

```diff
@@ -0,0 +1,102 @@
+      -  ./contrib/dist/make_dist_tarball
+
+## Ensure that the tarball actually works
+- [ ] Ensure openmpi `make distcheck` passes
+- Build and install Open MPI (from a tarball)
+  - [ ] Check that the version number and release date is correct in the installed man pages
+- [ ] Build and run the examples in the tarball
+  - [ ] MPI / C bindings
+  - [ ] MPI / C++ bindings
+  - [ ] MPI / All three Fortran bindings (you'll need newer compiler, e.g. gcc 4.9 or higher)
+  - [ ] OSHMEM / C bindings (don't run if not using ikrit)
+  - [ ] OSHMEM / Fortran bindings (don't run if not using ikrit)
+- [ ] Build and install the IBM tests from the `ompi-tests` repo
+  - [ ] Ensure the tests pass with at least 1 or 2 different MPI transports
+- [ ] With a prior release tarball from the same release series (if applicable):
+  - [ ] Build and install Open MPI
+  - [ ] Build all the examples in the tarball (against the prior OMPI release)
+  - [ ] Build the IBM tests from the `ompi-tests` repo (against the prior OMPI release)
+  - [ ] `rm -rf` the installation of the prior release
+  - [ ] Set `LD_LIBRARY_PATH` to point to the installation of the new Open MPI
+  - [ ] Run examples, make sure they still work
+  - [ ] Run IBM tests, make sure they still work
+
+## Test packaging
+- [ ] Ensure that building an Open MPI source RPM works on a RHEL system
+- [ ] Ensure that building the Open MPI binary RPM works on a RHEL system (note had to do hacking on buildrpm.sh script and it doesn't report correct information about where rpms are installed, at least not on my RHEL7.2 system running as root)
+- [ ] Install both RPMs on an RHEL system
+- [ ] Test building and running the IBM tests against the RPM-installed Open MPI
+- [ ] Test building a MPI-based program with the output from `pkg-config` with the RPM-installed `ompi.pc`
+- [ ] Test that uninstall of the rpm works
+
+## Do the release
+- [ ] Find the appropriate git hash to tag: look in the `VERSION` file in the official release tarball, and look at the `repo_rev` field.
+    - [ ] Make the release tag: `git tag -a vX.Y.Z HASH`
+    - [ ] Make the tag commit message be "Release vX.Y.Z"
+    - [ ] Double check that you tagged the Right commit, and that it exactly agrees with the hash in `VERSION` in the release tarball
+    - [ ] Triple check that you tagged the Right commit.  **TAGS ARE FOREVER!**
+- [ ] `git push --tags --dry-run` to the ompi repo.  Remove `--dry-run` when you're convinced it is correct.
+- Publish the tarballs on the Open MPI web site
+  - If Z is 0 (i.e., this is the first release in a series):
+    - [ ] `cp -r software/ompi/vCURRENT_RELEASE software/ompi/vRELEASE` (where RELEASE is for the new release X.Y and CURRENT_RELEASE is the X.Y of the current release)
+    - [ ] Edit each file in the new directory to update for the new release
+        - [ ] Edit `timeline-graph.php` to indicate relevant dates, such as branching
+    - [ ] Remove all prior tarballs from the `downloads` subdirectory
```

Comment:
Oh no - definitely don't do that! Remove only the rc's from this release. Leave all prior tarballs alone.